code/ConvertToStandardPath_SubmarineCable.py

```diff
@@ -254,21 +225,40 @@ def get_data_from_database(db_file):
             for path in convert_multilinestring_to_list(cable_wkt):
                 cableIdToWkt[cable_id].add(str(path))
 
-    for cable_id, _, city_name, city_latitude, city_longitude in datas:
+    for cable_id, _, city_name, city_state, city_country, city_latitude, city_longitude in datas:
         if cable_id in wktToCities:
             wktToCities[cable_id].add(
-                (city_name, city_latitude, city_longitude))
+                (city_name, city_state, city_country, city_latitude, city_longitude))
         else:
             wktToCities[cable_id] = set()
             wktToCities[cable_id].add(
-                (city_name, city_latitude, city_longitude))
+                (city_name, city_state, city_country, city_latitude, city_longitude))
     conn.close()
 
-    for cable_id, cable_wkt, _, _, _ in datas:
+    for cable_id, cable_wkt, _, _, _, _, _ in datas:
         cableIdToWkt[cable_id] = [ast.literal_eval(str(list_str)) for list_str in cableIdToWkt[cable_id]]
 
     return wktToCities, cableIdToWkt
 
+def list_to_linestring(coords_list):
```

Comment:
`coord_list_to_linestring` or `coords_to_linestring`?

Reply:
fixed to coord_list_to_linestring